src/scenes/home/getInvolved/successStories/successStories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Section from 'shared/components/section/section';
+import ImageListItem from 'shared/components/imageListItem/imageListItem';
+import image1 from 'images/Jon-Deng.jpg';
+import image2 from 'images/Jameel-Matin.jpg';
+import image3 from 'images/Sean-McBride.jpg';
+import content from './successStoriesContent.json';
+import styles from './successStories.css';
+
+const SuccessStories = () => (
+  <Section title="Success Stories" theme="white">
+    <div className={styles.successStoriesContent}>
+      <ImageListItem image={image1} title={content.items[0].title} cardText={content.items[0].body} />
```

Comment:
Once you've changed the content file to a `.js` file, you can use `.map` to spread the data across [like so](https://github.com/OperationCode/operationcode_frontend/blob/master/src/scenes/home/team/boardMembers.js)